pkg/notes/template.go

```diff
@@ -0,0 +1,250 @@
+package notes
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/template"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+)
+
+type Template struct {
+	UpgradeNotes [14]SIG
+	Metrics      Metrics
+	Features     Features
+	APIChanges   []string
+	Other        [14]SIG
+}
+
+type SIG struct {
+	Header  string
+	Content []string
+}
+
+var availableSIGs = map[string]definedSIG{
```

Comment:
Where did this list come from? There are [more than 14 SIGs](https://github.com/kubernetes/community/blob/master/sigs.yaml), even if the last few releases haven't seen release notes that fall under them.